api/app/Http/Controllers/Forms/FormController.php

```diff
@@ -186,6 +186,7 @@ public function duplicate($id)
         // Create copy
         $formCopy = $form->replicate();
         $formCopy->title = 'Copy of ' . $formCopy->title;
+        $formCopy->removed_properties = null;
```

Comment:
By default removed properties should be `[]` - I think null may cause some issues